packages/edit-site/src/components/dataviews/view-list.js

```diff
@@ -103,24 +103,25 @@ function HeaderMenu( { dataView, header } ) {
 									} }
 								>
 									{ info.label }
-								</DropdownMenuItemV2>
+								</DropdownMenuRadioItemV2Ariakit>
 							)
 						) }
-					</DropdownMenuGroupV2>
+					</DropdownMenuGroupV2Ariakit>
 				) }
 				{ isHidable && (
-					<DropdownMenuItemV2
+					<DropdownMenuItemV2Ariakit
 						prefix={ <Icon icon={ unseen } /> }
-						onSelect={ ( event ) => {
-							event.preventDefault();
+						onClick={ ( event ) => {
+							// what was this needed for?
+							// event.preventDefault();
```

Comment:
I couldn't understand why we needed the `preventDefault()` call here, since it looks like the menu would disappear anyway once clicking "Hide"

Reply:
Let's remove it and see if we find any error

Reply:
Without it, it would close the drop down menu. With the new version it's not needed, so it can be safely removed.